src/ViewInstance.js

```diff
@@ -289,10 +293,44 @@ class ViewInstance extends React.Component {
     this.setState({ isImportRecordModalOpened: false });
   }
 
+  toggleCopyrightModal = () => {
+    this.setState(prevState => ({ isCopyrightModalOpened: !prevState.isCopyrightModalOpened }));
+  };
+
   toggleFindInstancePlugin = () => {
     this.setState(prevState => ({ findInstancePluginOpened: !prevState.findInstancePluginOpened }));
   };
 
+  // Get all identifiers for all records
+  getIdentifiers = (data) => {
+    const { identifierTypesById } = data;
+    const { ISBN, ISSN } = indentifierTypeNames;
+    const records = this.props?.resources?.selectedInstance?.records;
+    if (!records) {
+      return null;
+    }
+    const allIdentifiers = [].concat(
```

Comment:
@AndrewIsh what's the purpose of `concat` here?

Reply:
Arghh, good spot. I suspect I was originally deep cloning the array then switched to `map`, but forgot to remove the `[].concat()` I'll fix that.